src/components/carousel/index.tsx

```diff
@@ -229,6 +222,14 @@ const Carousel: FC<Props> = (props) => {
             direction="next"
             fullScreen={!!fullScreen}
           />
+          {paginationType === PaginationType.Dot ? (
```

Comment:
is there a reason why this needs to be inside and the numbers variant outside? Can't this be solved with positioning?

Reply:
Nope, because of the relative position on the carousel...we want dots on the image.